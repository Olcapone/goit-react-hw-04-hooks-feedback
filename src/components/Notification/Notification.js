import s from "./Notification.module.css";

const Notification = ({ message }) => (<h2 className={s.title}>{message}</h2>);

export default Notification;
