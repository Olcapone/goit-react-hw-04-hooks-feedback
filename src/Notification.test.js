import React from 'react';
import { shallow } from 'enzyme';
import Notification from './components/Notification/Notification';


describe("Notification", () => {
  it("has props", () => {
    const wrapper = shallow(<Notification message="test " />);
    expect(wrapper.debug()).toMatchSnapshot();
  })

 it('has no props', () => {
  const wrapper = shallow(<Notification />);
  expect(wrapper.debug()).toMatchSnapshot();
  })
  
});
