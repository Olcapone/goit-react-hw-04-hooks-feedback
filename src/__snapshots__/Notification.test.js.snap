// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification has no props 1`] = `"<h2 className=\\"title\\" />"`;

exports[`Notification has props 1`] = `
"<h2 className=\\"title\\">
  test 
</h2>"
`;
